plasmapy/atomic/nuclear.py

```diff
@@ -217,7 +230,7 @@ def _total_charge(particles):
         for particle in particles:
             try:
                 total_charge += atomic_number(particle)
-            except ValueError:
+            except Exception:
```

Comment:
Are you sure catching generic `Exception`s here is the best way to do this? That's pretty broad, isn't it?

Reply:
Nope!  It's not, so I fixed this.  This is one of the complaints that PyCharm has been giving me since I very recently started using it. 😹